nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -19,6 +19,7 @@
           skip-tag: true
           wipe-workspace: true
           basedir: "nfs-ganesha-debian"
+
```

Comment:
a spacing here would seem consistent, not sure, you can always apply this on the fly and test it out